include/faker-cxx/helper.h

```diff
@@ -4,6 +4,7 @@
 #include <random>
 #include <vector>
 
+#include "common/generator.h"
```

Comment:
header files from `include` directory shouldn't include files from `src` directory, everything needed in `include` directory should be in that dir please